dotnet-authserver/src/TeacherIdentity.AuthServer/Models/Mappings/UserImportJobRowMapping.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace TeacherIdentity.AuthServer.Models.Mappings;
+
+public class UserImportJobRowMapping : IEntityTypeConfiguration<UserImportJobRow>
+{
+    public void Configure(EntityTypeBuilder<UserImportJobRow> builder)
+    {
+        builder.ToTable("user_import_job_rows");
+        builder.Property(r => r.UserImportJobId).IsRequired();
+        builder.Property(r => r.Id).HasMaxLength(UserImportJobRow.IdMaxLength).IsRequired();
+        builder.HasKey(r => new { r.UserImportJobId, r.RowNumber });
+        builder.Property(r => r.Errors).HasColumnType("varchar[]");
+    }
```

Comment:
Need to mark `RowNumber` as required?

Reply:
Does this create FK on `UserImportJobId`?

Reply:
Done both of these